### ISSUE TITLE
Present browser with UINavigationController - dismiss gesture and scroll issue

### ISSUE THREAD

**Issue by ptcmobilevnhub** [suzuki-0000/SKPhotoBrowser#327]
I found an issue when presenting the photo browser with UINavigationController, the scroll view has the wrong behavior and dismisses gestures disappear (swipe up | down).

```
let browser = SKPhotoBrowser(photos: images, initialPageIndex: indexPath.row)
browser.delegate = self
present(UINavigationController(rootViewController: browser), animated: true, completion: {})
```
![simulator screen shot - iphone x - 2018-07-23 at 16 21 33](https://user-images.githubusercontent.com/28531707/43068706-6ca475de-8e95-11e8-8058-9e7ba77fefbb.png)

![simulator screen shot - iphone x - 2018-07-23 at 16 21 23](https://user-images.githubusercontent.com/28531707/43068719-72bc26a6-8e95-11e8-9a0f-ddc558638718.png)

**Comment by ruixingchen** [comment]
I don't think this is a general purpose of the library, try to present SKPhotoBrowser directly.

**Comment by xiaoweiFive** [comment]
in viewWillAppear add
 if #available(iOS 11.0, *) {
            self.pagingScrollView.contentInsetAdjustmentBehavior = .never
        } else {
            // Fallback on earlier versions
        }


![image](https://user-images.githubusercontent.com/17330916/56074133-cffc0100-5ddf-11e9-9aed-63bba4a86acc.png)
